sdk/agrifood/azure-verticals-agrifood-farming/swagger/README.md

```diff
@@ -1,14 +1,12 @@
 ## Generate autorest code
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json
+input-file: C:\Users\bhkansag\bhargav-kansagara\azure-rest-api-specs-pr\specification\agrifood\data-plane\Microsoft.AgFoodPlatform\preview\2021-07-31-preview\agfood.json
```

Comment:
why use a local url?

PS:
we recommend use `require` instead of `input-file` and input the readme.md url in the spec repo. e.g.
```
require: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/agrifood/data-plane/readme.md
```